microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,62 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
 

```

Comment:
Rendered page for review:

https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-2153